book/chapter.css

```diff
@@ -117,13 +117,14 @@ ol.chapters {
   padding: 0;
 }
 
-code[class*="language-"] {
+code, code[class*="language-"] {
   font-family: Inconsolata, Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
 }
 
 pre {
   background: rgba(242, 242, 242, 0.5);
   padding: 20px;
+  overflow-x: scroll;
```

Comment:
Mentioned it in my general comment, but I think this rule created an extra line at the bottom of each pre box! Catastrophe!